src/encoded/schemas/ontology_term.json

```diff
@@ -0,0 +1,70 @@
+{
+    "title": "Ontology Term",
+    "id": "/profiles/ontology_term.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "required": [
+        "identifier"
+    ],
+    "anyOf": [
+        {
+            "required": [
+                "submission_centers"
+            ]
+        },
+        {
+            "required": [
+                "consortia"
+            ]
+        }
+    ],
+    "identifyingProperties": [
+        "accession",
+        "aliases",
+        "identifier",
+        "uuid"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/identifier"
+        },
+        {
+            "$ref": "mixins.json#/modified"
+        },
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/status"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/tags"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        }
+    ],
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "accession": {
+            "accessionType": "OT"
```

Comment:
Do you really want to accession Ontology Terms or Ontologies?  I see you are accessioning many of the items in your schemas.  Ontology maintainers are generally fairly good at having a stable ID and uri for ontology terms and the ontology itself.  Will you reaccession with every term update?  Well behaved ontologies have mechanisms for obsoletion as opposed to updates and managing this within the portal via accessioning seems potential overkill and potential additional work to maintain.

At least in 4DN accessions are meant to be permanent identifiers that will always point to the same item - even with the replacement mechanism that we use if a file needs to be retired you that provide a redirect if the old accession is used the old item is still accessible.

We do not have versioning though we have definitely considered it as a way to allow updates without needing to mint new accessions - something to consider especially for protocol and quality metric items?